Src/main.c

```diff
@@ -224,7 +224,7 @@ an settings option)
 #endif
 
 #define VERSION_MAJOR 2
-#define VERSION_MINOR 8
+#define VERSION_MINOR 12
```

Comment:
the PR needs to be rebased on current main branch, these defines no longer exist